File: app/src/main/java/com/github/fo2rist/mclaren/models/Circuit.java
package com.github.fo2rist.mclaren.models;

import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class Circuit implements Serializable {
    public String id; // e.g "austria"
    public String country; // e.g "AUS",
    public String track; // e.g "Melbourne Grand Prix Circuit",
    public String city; // e.g "Melbourne",
    public String name; // e.g "Australian Grand Prix"
    public int laps;
    public double length;
    public double distance;
    public String seasons; // e.g "2009–2016",
    @SerializedName("gp_held")
    public int gpHeld;
    @SerializedName("wiki_link")
    public String wikiLink;
}

File: app/src/main/java/com/github/fo2rist/mclaren/models/Circuit.kt
package com.github.fo2rist.mclaren.models

import com.google.gson.annotations.SerializedName

/**
 * Race Circuit's (Race Track) info model.
 * !Used as model for storage!
 */
data class Circuit(
    var id: String, // e.g "austria"
    var country: String, // e.g "AUS"
    var track: String, // e.g "Melbourne Grand Prix Circuit"
    var city: String, // e.g "Melbourne"
    /** Default grand prix name for a circuit. */
    @SerializedName("name")
    var gpName: String, // e.g "Australian Grand Prix"
    var laps: Int,
    var length: Double,
    var distance: Double,
    var seasons: String, // e.g "2009–2016",
    @SerializedName("gp_held")
    var gpHeld: Int,
    @SerializedName("wiki_link")
    var wikiLink: String
)
